tools/profiling/CMakeLists.txt

```diff
@@ -41,5 +41,42 @@ if(Graphviz_FOUND)
 
 endif(Graphviz_FOUND)
 
-Add_Subdirectory(python)
+if(CMAKE_CXX_COMPILER)
```

Comment:
The perfetto content seems to be downloaded in our source directory, and will confuse git and possibly us with new untracked files. Maybe we should have a 3rd-party directory where we keep our external dependencies.

Reply:
I have tested again, FetchContent_Declare / FetchContent_Populate download and expand the tar.gz file into the `${PROJECT_BINARY_DIR}/_deps/perfetto_source-src/` directory. 

The call to `  FetchContent_GetProperties(perfetto_source SOURCE_DIR PERFETTO_SOURCE_DIR)` a few lines after that allow us to find exactly where it is, but when I tested it is always in the build directory